R/connection.R

```diff
@@ -113,15 +118,58 @@ spark_version <- function(sc) {
 #'
 #' @export
 spark_version_from_home <- function(spark_home) {
-  releaseFile <- file.path(spark_home, "RELEASE")
-  releaseContents <- readLines(releaseFile)
-  version <- gsub("Spark | built.*", "", releaseContents[[1]])
+  versionAttempts <- list(
```

Comment:
Excellent! I think this should pretty well nail it for most installations (good that we have the fallback env variable though just in case)